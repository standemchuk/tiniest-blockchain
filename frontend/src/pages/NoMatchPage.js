import React from 'react'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'

const NoMatchPage = () => (
  <Grid container>
    <Grid item xs={12}>
      <Typography style={{ marginLeft: 15 }} variant={'display4'}>Not found</Typography>
    </Grid>
  </Grid>
)

export default NoMatchPage
