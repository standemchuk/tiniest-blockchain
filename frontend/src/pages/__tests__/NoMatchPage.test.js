import React from 'react'
import renderer from 'react-test-renderer'

import NoMatchPage from '../NoMatchPage'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <NoMatchPage />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
