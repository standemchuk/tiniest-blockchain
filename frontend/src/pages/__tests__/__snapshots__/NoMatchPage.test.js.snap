// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-container-1"
>
  <div
    className="MuiGrid-item-2 MuiGrid-grid-xs-12-40"
  >
    <h1
      className="MuiTypography-root-97 MuiTypography-display4-98"
      style={
        Object {
          "marginLeft": 15,
        }
      }
    >
      Not found
    </h1>
  </div>
</div>
`;
